config/shared/locales.json

```diff
@@ -1,6 +1,12 @@
 [
   { "code": "en", "label": "English" },
-  { "code": "cs", "label": "česky" },
+  { "code": "de", "label": "Deutsch" },
+  { "code": "es", "label": "español, castellano" },
```

Comment:
Which one to keep? Or both? We're trying to have the language name in its own language.

@Akronix what do you think?

Reply:
To avoid disputes and diving into language politics (usually impossible to resolve and there are _many_ as you'll surely learn in coming months), I just usually every project refer to whatever Wikipedia uses for their language menus.

They use just español so I'd go with that here too.

Reply:
Personally, I'd prefer "Castellano". But @simison has an interesting point and it's better to have a well-founded rule than relying on personal preferences. So I'd go for Español, also I feel it's more familiar for international people.

Reply:
I know the feeling, I also have preferences for certain ways to express some languages. That said, I've found it easier to be fair and consistent by relying on external well defined list that is relatively "neutral". UN list of languages is another good one.